libraries/Microsoft.Bot.Schema/ActivityEx.cs

```diff
@@ -157,6 +157,43 @@ public static ITraceActivity CreateTraceActivity(string name, string valueType =
             };
         }
 
+        /// <summary>
+        /// Createa a command to set recognition hints.
+        /// </summary>
+        /// <param name="expected">List of expected recognition hints.</param>
+        /// <param name="possible">List of possible recognition hints.</param>
+        /// <returns>New command activity.</returns>
+        public static ICommandActivity CreateRecognitionHints(IEnumerable<RecognitionHint> expected, IEnumerable<RecognitionHint> possible = null)
+        {
+            var hints = new List<RecognitionHint>();
+            foreach (var hint in expected)
+            {
+                var clone = hint.Clone();
+                clone.Importance = RecognitionHintImportance.Expected.ToString();
+                hints.Add(clone);
+            }
+
+            if (possible != null)
+            {
+                foreach (var hint in possible)
+                {
+                    var clone = hint.Clone();
+                    clone.Importance = RecognitionHintImportance.Possible.ToString();
+                    hints.Add(clone);
+                }
+            }
+
+            var reply = new Activity(ActivityTypes.Command)
+            {
+                Value = new CommandValue<List<RecognitionHint>>()
+                {
+                    CommandId = "recognitionHints",
```

Comment:
- Likely this should be the encoded in the Command Name. The command result will come back with the same name as the command. 
- Command Id is an optional tracking id should the bot needed to send multiple duplicate commands for the same action.